### PR TITLE
Update Tempto documentation

### PR DESCRIPTION
**Pull request by losipiuk**
Update Tempto documentation

### CODE REVIEW

**Comment by losipiuk** [comment]
@sopel39, @brian-rickman please take a look when you have some time.

File: README.md [hunk 1]
```diff
@@ -395,11 +395,60 @@ One can request that mutable table is in one of three states:
...
+                         + "where nation.region_key = region.key"))
+                   .hasAnyRows();
+    }
+```
+
+When tests are executed three instances of testJoinNationAndRegion test will be run.
+One for each of the requirements passed to `allOf` method. Note that there are three requirements
+passed to `allOf` method but each one is composite one and internally consists of two `mutableTable` requirements.
+Composition is performed using `compose` methods. The following requirements sets will be used:
+ * both `nation` and `region` tables storead as `TEXT`
```
Comment (brian-rickman) [inline README.md]: s/storead/stored/  (all three lines)

File: README.md [hunk 2]
```diff
@@ -579,11 +633,11 @@ Header parameters are:
 #### TABLE_NAME.data-revision
 
 Currently only HIVE table manager makes use of that. It should contain any string, which must be updated when
-table contents is changed.
+table contents is changed. It is used for determining if resending table data to HDFS cluster is required or not.
 
 ### Tests
 
-Test case files are stores in `sql-tests/testcases_ directory`. The directory right under the
+Test case files are are stored in `sql-tests/testcases_ directory`. The directory right under the
```
Comment (brian-rickman) [inline README.md]: s/are are/are/

File: README.md [hunk 3]
```diff
@@ -657,7 +708,7 @@ TODO more info on scripts, what they should be named, what they can contain.
 
 #### Using tables across databases.
 
-It is possible (which is useful for testing presto for example), to use a table which is created in one database (e.g. hive, psql)
+It is possible (which is useful for testing Presto for example) to use a table which is created in one database (e.g. hive, psql)
 while sending test query to other database (e.g. presto).
```
Comment (brian-rickman) [inline README.md]: s/presto/Presto/

File: README.md [hunk 4]
```diff
@@ -657,7 +708,7 @@ TODO more info on scripts, what they should be named, what they can contain.
...
 Take a look at the example below. Here query is issued via presto JDBC, while nation table could be created somewhere else. In order to determine
```
Comment (brian-rickman) [inline README.md]: s/presto/Presto/

**Comment by brian-rickman** [comment]
Couple minor corrections but overall LGTM.

**Pull request merged**